source/includes/_policies.md

```diff
@@ -0,0 +1,91 @@
+# Policies
+
+## /v2/policy/
+
+Note: this endpoint is currently in beta. The contract should not be considered stable, and is subject to change without notice.
```

Comment:
Can we not use _contract_ to indicate the data, "The contract should not be considered stable" is not a friendly or simple way of explaining the fact that the POST data expected and the response data may change.

Reply:
Done